packages/core-data/src/resolvers.js

```diff
@@ -54,7 +54,7 @@ export function* getCurrentUser() {
 export function* getEntityRecord( kind, name, key = '' ) {
 	const entities = yield getKindEntities( kind );
 	const entity = find( entities, { kind, name } );
-	if ( ! entity ) {
+	if ( entity ) {
```

Comment:
seehttps://github.com/WordPress/gutenberg/issues/24308

Reply:
this change is problematic for me. `entity` will always be defined aside on the very first call so `getEntityRecord`  won't do anything for all calls except the first one?

Reply:
@youknowriad I replied in https://github.com/WordPress/gutenberg/issues/24308

Reply:
To close the loop here - this change should **not** be merged, it just happens to be useful on the widget screen exclusively until #19498 lands.